static/js/pivotal.js

```diff
@@ -8,6 +8,12 @@
     }
   };
 
+  // register common error handler on Ajax errors / failures
+  function logError(err) {
```

Comment:
Maybe you can inline this method, since it's not used anywhere else.
If you think the `logError` name is useful, you can write:
```javascript
$(document).ajaxError(function logError(err) {
  console.error(err)
});
```
Otherwise, this is enough:
```javascript
$(document).ajaxError(console.error);
```